api/opds_for_distributors.py

```diff
@@ -398,7 +393,7 @@ def run_once(self, progress):
             .join(Identifier)
             .filter(LicensePool.collection_id == self.collection.id)
             .filter(~Identifier.id.in_(identifier_ids))
-            .filter(LicensePool.licenses_available > 0)
+            .filter(LicensePool.licenses_available == LicensePool.UNLIMITED_ACCESS)
```

Comment:
This won't work for existing content without a database migration.

Reply:
Working on this now. :+1: